test/utils/runTypeScriptFile.ts

```diff
@@ -4,7 +4,7 @@ export async function runTypeScriptFile(scriptPath: string): Promise<void> {
   const childProcess = child_process.exec(`ts-node ${scriptPath}`);
   await new Promise((resolve, reject) => {
     childProcess.on("exit", (code) => {
-      const SUCCESSFUL_EXIT_CODE = 0;
+      const SUCCESSFUL_EXIT_CODE = 1;
```

Comment:
"1" means that there was an error, not a successful exit. To achieve the successful exit you have to fix the DiscordBadge to take props as described above and add it into the `examples/README.md.tsx` file.